src/apps/Console/Console.js

```diff
@@ -243,8 +200,41 @@ function Console({ apps, wrapper }) {
         setLoading(false)
       }
     },
-    [wrapper, performIntents, web3, toast]
+    [wrapper, performIntents, toast, web3]
   )
+
+  // handle input change
+  function handleChange(input) {
```

Comment:
Does the linter warn if we wrap this around a `useCallback` with no dependencies? We currently re-create this function on every render, when creating it just once would be enough.

Reply:
No warnings; memoized ✅